makefiles/host/assemble.mk

```diff
@@ -9,9 +9,9 @@ include rootfs.mk
 SHELL := /bin/bash
 
 # define the layout of the image
-IMGSIZE = 4096MiB
```

Comment:
This is a bit unfortunate, as it no longer fits 4GB sdcards. If this is necessary me might aswell grow to 8GB, no?